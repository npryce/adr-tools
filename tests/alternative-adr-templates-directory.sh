mkdir templates-dir
echo "Hello template" > templates-dir/template.md

ADR_TEMPLATES_DIR=templates-dir
export ADR_TEMPLATES_DIR

adr new Example

head -1 doc/adr/0001-example.md

