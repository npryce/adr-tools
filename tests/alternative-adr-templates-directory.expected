+ mkdir templates-dir
+ echo 'Hello template'
+ ADR_TEMPLATES_DIR=templates-dir
+ export ADR_TEMPLATES_DIR
+ adr new Example
doc/adr/0001-example.md
+ head -1 doc/adr/0001-example.md
Hello template
